style.css

```diff
@@ -1,10 +1,113 @@
-body {
-background-color: grey;
-font-family: "GothamPro";
+ * {
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/step/pullRequest?prid=5930829)